docs/DeveloperGuide.md

```diff
@@ -499,6 +514,125 @@ java -jar PropertyRentalManager.jar
 ### Delete
 
 ### List
+1. List clients with or without tags
+   1. Test case `list -client`  
+      Expected: All clients all listed with all their details. The format of details of each client is given below -  
+      ```
+      1. Client Name: Doja Cat
+         Client Contact Number: 93437878
+         Client Email: doja88@example.com
+         Client Budget: 2000
```

Comment:
Usually when we print out message for the users, we want it to be clear. That's why when the addition of client is successful, there's a message showing the currency.

Would be highly recommended that all your printing message with regards to money includes a currency.